File: CS/WinWebSolution.Module.Web/WebSchedulerFilterResourcesListViewController.cs
using DevExpress.ExpressApp;
using DevExpress.Data.Filtering;
using DevExpress.Xpo;

namespace WinWebSolution.Module {
    public class WebSchedulerFilterResourcesListViewController : SchedulerFilterResourcesListViewControllerBase {
        protected override void FilterResourcesDataSource(object resourcesDataSource, CriteriaOperator criteria) {
            if (resourcesDataSource is XpoDataSource) {
                ((XpoDataSource)resourcesDataSource).Criteria = criteria.ToString();
            }
        }
    }
}

File: VB/WinWebSolution.Module.Web/WebSchedulerFilterResourcesListViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.Data.Filtering
Imports DevExpress.Xpo

Namespace WinWebSolution.Module
	Public Class WebSchedulerFilterResourcesListViewController
		Inherits SchedulerFilterResourcesListViewControllerBase
		Protected Overrides Sub FilterResourcesDataSource(ByVal resourcesDataSource As Object, ByVal criteria As CriteriaOperator)
			If TypeOf resourcesDataSource Is XpoDataSource Then
				CType(resourcesDataSource, XpoDataSource).Criteria = criteria.ToString()
			End If
		End Sub
	End Class
End Namespace